draft-foudil-securitytxt.md

```diff
@@ -366,7 +366,7 @@ Web-based services SHOULD place the security.txt file under the /.well-known/ pa
 as per {{!RFC5785}}. A security.txt file located under the top-level path SHOULD either redirect (as per section 6.4 of {{!RFC7231}})
 to the security.txt file under the /.well-known/ path or be used as a fallback if the ".well-known" path cannot be used.
 
-If retrieval of a "security.txt" file results in a redirect (as per
+If retrieval of a "security.txt" file from the top-level path results in a redirect (as per
 section 6.4 of {{!RFC7231}}), the implementors MUST NOT follow
 redirects that lead to another domain or subdomain
 but SHOULD follow redirects within the same domain name
```

Comment:
If there is a circumstance where the top-level path redirects externally, and so does the .well-known path, then I think even though we can't trust the top-level path redirect we can nonetheless continue to trust the .well-known redirect.

This isn't the only reasonable interpretation of the amended text because one might think that a non-compliant redirect is an indicator of malicious activity.

I propose as follows: swap out "MUST NOT follow redirects that" for "MUST NOT follow that redirect if it" (and of course change the rest of the sentence to fit, and do the same for the "SHOULD".

Reply:
done